client-app/modules/news/pages/news-article.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <div class="news-article-page">
+  <div class="news-article">
     <Error404 v-if="!loading && !newsArticle"></Error404>
```

Comment:
Please use self-closing tags (<Error404 v-if="!loading && !newsArticle" />) for components. Here and below.